src/NewTwitchApi/NewTwitchApi.php

```diff
@@ -14,15 +14,47 @@
 
 class NewTwitchApi
 {
-    private $oauthApi;
-    private $gamesApi;
-    private $streamsApi;
-    private $usersApi;
-    private $webhooksApi;
-    private $webhooksSubscriptionApi;
-
-    public function __construct(Client $helixGuzzleClient, string $clientId, string $clientSecret, Client $authGuzzleClient = null)
+    /**
+     * @var OauthApi
```

Comment:
I don't think we need these `@var` annotations since the constructor initializes these properties which tells us what we need to know. However, while we want for the ability to type-hint these properties, I'm ok using `@var` to be explicit here.

Reply:
This is to help the IDE along & remain descriptive so that the user knows what to expect in this variable

Reply:
Yes, this assists certain IDE's. VS Code, for example, benefits from this.

Reply:
I'm so used to IntelliJ's IDEs that I forget not all IDEs/editors work as well. :-) Like I said before, I'm fine leaving these in.